docs/source/dev/environment.md

```diff
@@ -57,8 +57,9 @@ conda env create -f conda-environment.yml --name another_name
 To choose a specific Python version when creating the conda environment,
 for example Python 3.9, one can use a command such as the following:
 
-```none
-conda env create -f conda-environment.yml python=3.9
+```
+conda create python=3.9 --name mxcubeweb
+conda env update --name mxcubeweb --file conda-environment.yml
```

Comment:
The [issue ticket](https://github.com/mxcube/mxcubeweb/issues/1380#issue-2492532184) says that this technique does not work. Or did I misunderstand something?

Reply:
This should work, however we cant pass `python=` to `conda env create` like `conda env create -f conda-environment.yml python=3.9` its not valid syntax.

Then there  might be some other dependency issue, but that is a sepereate problem

Reply:
OK then I suggest using Python 3.10 as an example. It seems is should resolve. At least it does resolve with `micromamba create -f conda-environment.yml 'python==3.10'`.

```suggestion
conda create python=3.10 --name mxcubeweb
conda env update --name mxcubeweb --file conda-environment.yml
```

Reply:
And then if we have an issue with conda not being able to resolve for Python 3.8 or Python 3.9, then we should address it. I do not see much point in fixing the documentation by writing something that we already know does not deliver the expected result.

That is why I always prefer we solve one issue at a time. This pull request's title is about Redis, so I'd recommend we handle only Redis here and look at the rest of the issues one after the other in separate issues/PRs.

Reply:
Ok

Reply:
https://github.com/mxcube/mxcubeweb/issues/1383